### DISCUSSION TITLE
Unable to get MIDI2LR to boot/work with LR CC.

### DISCUSSION THREAD

**Discussion by zacharyulibarri** [rsjaffe/MIDI2LR#773] [Problems]
Title says it all. It is incredibly frustrating that Adobe chose to name one version LR CC and the other one LRC. Googling for these problems just leads me to stuff about LRC.

Anywho, I can't get MIDI2LR to work with CC. The documentation seems to suggest that it should work, but it refuses to launch with CC. I can get it work with Classic, but even with that window open it does nothing to the CC window.

Is this unexpected behavior? I bought the XTouch Mini exclusively for this purpose. I'm going to be quite bummed if it doesn't work.

(and just for clarification, I have no desire to go back to LRC. The ability to make edits on one device and then have them automatically on my phone is something that is now a major part of my workflow.)

Running the most recent 6.0.1.0.

**Comment by 153957** [comment]
> MIDI2LR is an application that interfaces MIDI controllers with Lightroom Classic. Version 4 requires Lightroom 10 Classic, version 5 Lightroom 11 Classic: use version 3 for earlier versions of Lightroom.

> Prerequisites
> Lightroom 6+ or Lightroom CC Classic. Version 4 requires Lightroom 10 Classic, version 5 Lightroom 11 Classic: use version 3 for earlier versions of Lightroom.

The 'CC' in that last paragraph refers to 'Creative Cloud' which Lightroom Classic is also part of, it does not indicate it is the 'Cloud' version of Lightroom

So unfortunately, I dont think it will work.
